.goreleaser.yml

```diff
@@ -52,6 +52,34 @@ before:
         ".releaseSeries |= (. + [{contract: \"v1beta1\", major: {{ .Major }}, minor: {{ .Minor }}}] | unique)" \
         metadata.yaml >release-metadata.yaml'
     - make template-helm-repository
+    # dev image
+    - |
+      sh -ec 'if [ {{ .IsSnapshot }} == true ] ; then
+        GOOS=linux \
+        SOURCE_DATE_EPOCH=$(date +%s) \
+        KO_DOCKER_REPO=ko.local/{{ .ProjectName }} \
+          ko build \
+          --bare \
+          --tags v{{ trimprefix .Version "v" }} \
+          ./cmd
+      fi'
+    # released image
+    - |
+      sh -ec 'if [ {{ .IsSnapshot }} == false ] ; then
+        unset GOOS GOARCH && \
```

Comment:
ko complained about these envs being set together with the `--platform` flag.

Reply:
I think this is the ultimate problem btw but I couldn't figure out why or how it made a difference 🤔

Reply:
https://github.com/ko-build/ko/issues/274